docs/modules/setup/pages/diagram-options.adoc

```diff
@@ -62,7 +62,12 @@ With flag set, option takes effect.
 * `terminal` (`300`)
 * `terminal-grayscale` (`301`)
 
-| See: https://d2lang.com/tour/themes/
+|See: https://d2lang.com/tour/themes/
+
+|sketch
+|_boolean_ +
```

Comment:
I think we should use a flag (similar to `antialias` and `no-transparency` from Blockdiag) rather than a boolean.

```
GET https://kroki.io/d2/svg/base64encodeddiagram?sketch
```